feat: Add codejail service

timmc-edx opened the pull request:
The image specified here is not yet available, but should be the right name for after <https://github.com/edx/public-dockerfiles/pull/85> merges. I've tested with a local tag.

The codejail-service repo will be pulled by default but will not be a dependency of lms/cms (yet?).

This is part of https://github.com/edx/edx-arch-experiments/issues/894

----

I've completed each of the following or determined they are not applicable:

- [x] Made a plan to communicate any major developer interface changes (or N/A)

jmbowman commented:
We've added a newer database dump in the meantime, and are getting close to a Django upgrade that should make the migrations much faster.

timmc-edx committed:
feat: Add codejail service

The image specified here is not yet available, but should be the right
name for after <https://github.com/edx/public-dockerfiles/pull/85>
merges. I've tested with a local tag.

This is part of https://github.com/edx/edx-arch-experiments/issues/894
